matplotcheck/raster.py

```diff
@@ -57,14 +57,15 @@ def assert_colorbar_range(self, crange):
             ), "Colorbar maximum is not expected value:{0}".format(crange[1])
 
     def _which_label(self, label, all_label_options):
-        """Helper function for assert_legend_accuracy_classified_image
+        """Helper function for assert_legend_labels
         Returns string that represents a category label for label.
 
         Parameters
         ----------
         label: string from legend to see if it contains an option in
```

Comment:
this is confusing. i think label is  a single label found in the input plot legend.